src/components-examples/material/autocomplete/autocomplete-harness/autocomplete-harness-example.spec.ts

```diff
@@ -14,7 +14,9 @@ describe('AutocompleteHarnessExample', () => {
   let loader: HarnessLoader;
 
   beforeAll(() => {
-    TestBed.initTestEnvironment(BrowserDynamicTestingModule, platformBrowserDynamicTesting());
+    TestBed.initTestEnvironment(BrowserDynamicTestingModule, platformBrowserDynamicTesting(), {
+      teardown: {destroyAfterEach: true}
```

Comment:
Can we also enable `rethrowErrors` everywhere as well?

```suggestion
      teardown: {destroyAfterEach: true, rethrowErrors: true}
```

Reply:
It default to rethrowing when `destroyAfterEach` is enabled: https://github.com/angular/angular/blob/master/packages/core/testing/src/r3_test_bed.ts#L482